### ISSUE TITLE
Custom scalar examples from metadata.json no longer used in 1.0.5

### ISSUE THREAD

**Issue by simoncoggins** [anvilco/spectaql#279]
Our schema has a number of custom scalars such as:
```
scalar param_integer
```

In 0.15.0 we were able to define examples for them in a metadata.json file:

```
{
  "SCALAR": {
    "param_integer": {
      "documentation": { "example": "123" }
    }
  }
}
```
and have that show up under the scalar description example in the documentation.

In version 1.0.5 that is no longer working - the same schema file and metadata.json description no longer results in the provided example (e.g. the example displays 'param_integer' not '123'):

<img width="755" alt="Screen Shot 2022-04-05 at 8 54 56 PM" src="https://user-images.githubusercontent.com/123462/161717564-14b42f67-e97e-42a0-b3a6-6e818ffe403d.png">

Unrelated, but in 0.15.0, even though the above example works, I noticed that when custom scalars are used as query arguments the default example for the query doesn't pull through the custom scalar example data like it does for existing scalars. It would be fantastic if it did!

**Comment by newhouse** [comment]
Should be fixed in `1.0.8`.

**Comment by simoncoggins** [comment]
Confirmed this is fixed (tested in 1.0.9). Custom scalar docs are now showing and the second issue (using custom scalar examples when the type is used elsewhere) seems to be working too, which is awesome, thanks!
